src/GraphQL/Types/Composite/AutoRegisteringHelper.cs

```diff
@@ -10,15 +10,17 @@ namespace GraphQL.Types
 {
     internal static class AutoRegisteringHelper
     {
-        internal static void SetFields<TSourceType>(ComplexGraphType<TSourceType> type, IEnumerable<PropertyInfo> properties, params Expression<Func<TSourceType, object?>>[]? excludedProperties)
+        internal static IEnumerable<PropertyInfo> ExcludeProperties<TSourceType>(IEnumerable<PropertyInfo> properties, params Expression<Func<TSourceType, object?>>[]? excludedProperties)
```

Comment:
Why do you need to split code into two methods?

Reply:
Read the next PR that contains migration doc changes.

Reply:
OK